docs/faq.md

```diff
@@ -116,6 +118,38 @@ techs made available to video.js. For example, video.js 5 includes the Flash tec
 enables the playback of FLV video where the Flash plugin is available. For more information
 on media formats see the [troubleshooting guide][troubleshooting].
 
+## Q: How to I autoplay the video?
+
+Video.js supports the standard html5 `autoplay` attribute on the video element.
+It also supports it as an option to video.js or as a method invocation on the player.
+
+```html
+<video autoplay controls class="video-js">
+```
+
+```js
+var player = videojs('my-video', {
+  autoplay: true
+});
+
+// or
+
+player.autoplay(true);
+```
+
+### Q: How can I autoplay a video on a mobile device?
+
+Most mobile devices have blocked autoplaying videos until recently.
+For mobile devices that don't support autoplaying, there's nothing that can be done.
```

Comment:
", autoplay is not supported by Video.js."

"nothing that can be done" invites digression into canvas hacks and the like.